Mathlib/Analysis/NormedSpace/Exponential.lean

```diff
@@ -447,153 +472,161 @@ theorem expSeries_summable (x : 𝔸) : Summable fun n => expSeries 𝕂 𝔸 n
   (norm_expSeries_summable x).of_norm
 #align exp_series_summable NormedSpace.expSeries_summable
 
-theorem expSeries_summable' (x : 𝔸) : Summable fun n => (n !⁻¹ : 𝕂) • x ^ n :=
-  (norm_expSeries_summable' x).of_norm
+variable (𝕂) in
+theorem expSeries_summable' [Algebra ℚ 𝔸] (x : 𝔸) : Summable fun n => (n !⁻¹ : ℚ) • x ^ n :=
+  (norm_expSeries_summable' 𝕂 x).of_norm
 #align exp_series_summable' NormedSpace.expSeries_summable'
 
-theorem expSeries_hasSum_exp (x : 𝔸) : HasSum (fun n => expSeries 𝕂 𝔸 n fun _ => x) (exp 𝕂 x) :=
+variable [Algebra ℚ 𝔸] [Algebra ℚ 𝔹]
+
+theorem expSeries_hasSum_exp (x : 𝔸) : HasSum (fun n => expSeries 𝕂 𝔸 n fun _ => x) (exp x) :=
   expSeries_hasSum_exp_of_mem_ball x ((expSeries_radius_eq_top 𝕂 𝔸).symm ▸ edist_lt_top _ _)
 #align exp_series_has_sum_exp NormedSpace.expSeries_hasSum_exp
 
-theorem exp_series_hasSum_exp' (x : 𝔸) : HasSum (fun n => (n !⁻¹ : 𝕂) • x ^ n) (exp 𝕂 x) :=
+theorem exp_series_hasSum_exp' (x : 𝔸) : HasSum (fun n => (n !⁻¹ : 𝕂) • x ^ n) (exp x) :=
   expSeries_hasSum_exp_of_mem_ball' x ((expSeries_radius_eq_top 𝕂 𝔸).symm ▸ edist_lt_top _ _)
 #align exp_series_has_sum_exp' NormedSpace.exp_series_hasSum_exp'
 
-theorem exp_hasFPowerSeriesOnBall : HasFPowerSeriesOnBall (exp 𝕂) (expSeries 𝕂 𝔸) 0 ∞ :=
+theorem exp_hasFPowerSeriesOnBall : HasFPowerSeriesOnBall exp (expSeries 𝕂 𝔸) 0 ∞ :=
   expSeries_radius_eq_top 𝕂 𝔸 ▸ hasFPowerSeriesOnBall_exp_of_radius_pos (expSeries_radius_pos _ _)
 #align exp_has_fpower_series_on_ball NormedSpace.exp_hasFPowerSeriesOnBall
 
-theorem exp_hasFPowerSeriesAt_zero : HasFPowerSeriesAt (exp 𝕂) (expSeries 𝕂 𝔸) 0 :=
+theorem exp_hasFPowerSeriesAt_zero : HasFPowerSeriesAt exp (expSeries 𝕂 𝔸) 0 :=
   exp_hasFPowerSeriesOnBall.hasFPowerSeriesAt
 #align exp_has_fpower_series_at_zero NormedSpace.exp_hasFPowerSeriesAt_zero
 
+section
+
 @[continuity]
-theorem exp_continuous : Continuous (exp 𝕂 : 𝔸 → 𝔸) := by
+theorem exp_continuous : Continuous (exp : 𝔸 → 𝔸) := by
   rw [continuous_iff_continuousOn_univ, ← Metric.eball_top_eq_univ (0 : 𝔸), ←
     expSeries_radius_eq_top 𝕂 𝔸]
   exact continuousOn_exp
 #align exp_continuous NormedSpace.exp_continuous
 
-theorem exp_analytic (x : 𝔸) : AnalyticAt 𝕂 (exp 𝕂) x :=
+end
+
+theorem exp_analytic (x : 𝔸) : AnalyticAt 𝕂 exp x :=
   analyticAt_exp_of_mem_ball x ((expSeries_radius_eq_top 𝕂 𝔸).symm ▸ edist_lt_top _ _)
 #align exp_analytic NormedSpace.exp_analytic
 
+variable (𝕂)
```

Comment:
Why do we need this?  If it's so that we pull in a NormedAlgebra assumption, couldn't we always do that over the reals (since complex normed algebras are real normed algebras)?